### ISSUE TITLE
Android app only shows "Loading..."

### ISSUE THREAD

**Issue by schrauger** [bdjett/disney-world-for-pebble#1]
With Android, the "Loading..." screen shows up when selecting any item (other than the built-in menu items).

There is at least one other person with the same issue, and they created an issue in the git repo of the previous version of this app. https://github.com/bdjett/my-disney-experience-for-pebble/issues/1

**Comment by scummboy** [comment]
I get this same problem. Using an Android phone as well.

**Comment by sjmatta** [comment]
I fixed this but haven't heard back from the author about the pull request yet.

**Comment by schrauger** [comment]
@sjmatta Thanks for the update. I started to troubleshoot the problem over a year ago and had some progress, but I lost steam and gave up after a while. Hopefully @bdjett will accept your pull request.

I'll try building the pebble app from your code, too, and see if it works. Though I recently updated my firmware to 2.0; is this app forward-compatible?

**Comment by sjmatta** [comment]
@schrauger Yeah, the pull request works in 2.0. Also, I started adding timeline support (see the branch). It works for dining reservations. I was going to add Fastpass reservations and then open another pull request but same as you- I ran out of steam! I will see if I can wrap it up soon.

**Comment by bdjett** [comment]
Sorry, the project has been kind of dead since I haven't had the time to work on it. More than glad to accept the pull request. Thanks @sjmatta!

**Comment by sjmatta** [comment]
Awesome thank you @bdjett!
